README.md

```diff
@@ -70,16 +89,52 @@ Resulting Bech32m address: bcrt1pe8q322zksrnd40lavu0gqu53f9a255gfah78gdupgeuu92j
 ```
 
 # 4 send token to script bech32m address
-bcli sendtoaddress bcrt1pe8q322zksrnd40lavu0gqu53f9a255gfah78gdupgeuu92jzn0ysnry0c9 0.001
+
+You can do the same on the console ui
+
+```bash
+sendtoaddress bcrt1pe8q322zksrnd40lavu0gqu53f9a255gfah78gdupgeuu92jzn0ysnry0c9 0.001
+```
+
+or using CLI
+
+```bash
+bcli -rpcuser=rpcuser -rpcpassword=rpcpassword sendtoaddress bcrt1pe8q322zksrnd40lavu0gqu53f9a255gfah78gdupgeuu92jzn0ysnry0c9 0.001
```

Comment:
bcli is alias point to bitcoin-cli , no need "-rpcuser=rpcuser -rpcpassword=rpcpassword"

alias bcli='bitcoin-cli -regtest -rpcuser=rpcuser -rpcpassword=rpcpassword -rpcport=18332' 

i miss this in environmental preparation

Reply:
got ya. I thought you renamed executable or something like that.Will edit accordingly